lib/money-rails/active_record/monetizable.rb

```diff
@@ -55,6 +55,16 @@ def monetize(*fields)
                                    "Use :as option to explicitly specify the name or change the amount column postfix in the initializer."
             end
 
+            # Assigns infinite_precision based on the field's column type, if available
+            unless options.has_key?(:infinite_precision)
+              if MoneyRails.infer_precision?
+                column_definition = self.columns_hash[subunit_name]
+                options[:infinite_precision] = column_definition && column_definition.type == :decimal
+              else
+                options[:infinite_precision] = MoneyRails.default_infinite_precision
```

Comment:
Is there any difference in behavior between this line, and not sending a value for the option at all?
If no option is specified, wont `Money` default to `Money.default_infinite_precision`?

Reply:
Yeah, you're right.  It is unnecessary since `MoneyRails.default_infinite_precision` delegates to `Money.default_infinite_precision`.  If it were different then it would be needed, but it is not.

Reply:
Removed unnecessary assignment.